platform/viewer/src/App.js

```diff
@@ -118,10 +119,12 @@ class App extends Component {
       <AppContext.Provider value={config}>
         <Provider store={store}>
           <I18nextProvider i18n={i18n}>
-            <Router basename={this.props.routerBasename}>
-              <WhiteLabellingContext.Provider value={this.props.whiteLabelling}>
+            <Router basename={routerBasename}>
+              <WhiteLabellingContext.Provider value={whiteLabelling}>
                 <SnackbarProvider>
-                  <OHIFStandaloneViewer />
+                  <ModalProvider modal={Modal}>
+                    <OHIFStandaloneViewer />
+                  </ModalProvider>
```

Comment:
I originally meant something like this:

```jsx
<ModalProvider>
    <Modal />
    <OHIFStandaloneViewer />
</ModalProvider>
```

But I think your implementation hammers home that there should be a single `Modal` component. Let's keep what you have 👍

Reply:
Done! thx.